src/Microsoft.DotNet.Build.CloudTestTasks/Microsoft.DotNet.Build.CloudTestTasks.csproj

```diff
@@ -25,6 +25,7 @@
   <ItemGroup>
     <Compile Include="CreateAzureContainer.cs" />
     <Compile Include="CreateAzureFileShare.cs" />
+    <Compile Include="GenerateAssemblyList.cs" />
```

Comment:
Can this task run in .Net Core? if yes, then perhaps we want to move it back to Microsoft.DotNet.Build.Tasks.csproj. This project is more intended to hold only the tasks that are not able to be converted yet due to external dependencies that are not in .NetCore yet.